From 785f64b8bbe9ee2e6c1c4e8eccb5a98ed7461d5a Mon Sep 17 00:00:00 2001
From: JP Abgrall <jpa@google.com>
Date: Fri, 22 Nov 2013 17:29:57 -0800
Subject: [PATCH] android: configs: require TCPMSS, remove SCHED_TRACER and
 TIMER_STATS

TCPMSS is required for the Android Vpn service to correctly
handle the MTU on tun/ppp devices.  Bug: 11579326
We don't really need SCHED_TRACER and the TIMER_STATS.

Change-Id: I10c5767a6324a496713752d4fe9eff361dc8e06a
(cherry picked from commit 23f01e8e81f3c53985958fa291b39c84293ad047)
---
 android/configs/android-base.cfg        | 1 +
 android/configs/android-recommended.cfg | 2 --
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/android/configs/android-base.cfg b/android/configs/android-base.cfg
index 2516202..a959765 100644
--- a/android/configs/android-base.cfg
+++ b/android/configs/android-base.cfg
@@ -85,6 +85,7 @@ CONFIG_NETFILTER_XT_TARGET_CONNMARK=y
 CONFIG_NETFILTER_XT_TARGET_MARK=y
 CONFIG_NETFILTER_XT_TARGET_NFLOG=y
 CONFIG_NETFILTER_XT_TARGET_NFQUEUE=y
+CONFIG_NETFILTER_XT_TARGET_TCPMSS=y
 CONFIG_NETFILTER_XT_TARGET_TPROXY=y
 CONFIG_NETFILTER_XT_TARGET_TRACE=y
 CONFIG_NET_CLS_ACT=y
diff --git a/android/configs/android-recommended.cfg b/android/configs/android-recommended.cfg
index b0120f6..1bd9f6f 100644
--- a/android/configs/android-recommended.cfg
+++ b/android/configs/android-recommended.cfg
@@ -95,7 +95,6 @@ CONFIG_PM_RUNTIME=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 CONFIG_POWER_SUPPLY=y
 CONFIG_SCHEDSTATS=y
-CONFIG_SCHED_TRACER=y
 CONFIG_SMARTJOYPLUS_FF=y
 CONFIG_SND=y
 CONFIG_SOUND=y
@@ -106,7 +105,6 @@ CONFIG_TABLET_USB_GTCO=y
 CONFIG_TABLET_USB_HANWANG=y
 CONFIG_TABLET_USB_KBTAB=y
 CONFIG_TABLET_USB_WACOM=y
-CONFIG_TIMER_STATS=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
 CONFIG_UHID=y
-- 
1.8.0

